includes/core/classes/class-event.php

```diff
@@ -104,20 +102,11 @@ public function __construct( int $post_id ) {
 	 * @throws Exception If date/time formatting fails or settings cannot be retrieved.
 	 */
 	public function get_display_datetime(): string {
-		$user_id     = get_current_user_id();
 		$settings    = Settings::get_instance();
-		$date_format = $settings->get_value( 'general', 'formatting', 'date_format' );
-		$time_format = $settings->get_value( 'general', 'formatting', 'time_format' );
+		$date_format = apply_filters( 'gatherpress_user_set_date_format', $settings->get_value( 'general', 'formatting', 'date_format' ) );
+		$time_format = apply_filters( 'gatherpress_user_set_time_format', $settings->get_value( 'general', 'formatting', 'time_format' ) );
```

Comment:
sorry, not these filters should stay the same. i just meant the methods. `gatherpress_ddate_format` and `gatherpress_time_format` are good names. sorry for the confusion.